restapi/src/main/java/org/cristalise/restapi/ScriptUtils.java

```diff
@@ -99,7 +99,7 @@ public Response executeScript(HttpHeaders headers, ItemProxy item, String script
             }
             catch (Exception e) {
                 Logger.error(e);
-                throw ItemUtils.createWebAppException("Error executing script, please contact support", e, Response.Status.NOT_FOUND);
+                throw ItemUtils.createWebAppException(e.getMessage(), e, Response.Status.NOT_FOUND);
```

Comment:
Sorry, I changed my mind. Let's check if the result of `e.getMessage()` contains an `[errorMessage]` tag: if it does, then let's use `e.getMessage()`, otherwise use the hardcoded text. Basically, if we had a user-friendly error message thrown from the script, then we cascade it further, otherwise we show the generic error, because it's more user-friendly than some technical exception message.

Reply:
Added if condition in the catch block.